internal/cypress/docker/runner.go

```diff
@@ -71,6 +75,28 @@ func (r *Runner) RunProject() (int, error) {
 	return errorCount, nil
 }
 
+// checkCypressVersion do several checks before running Cypress tests.
+func (r *Runner) checkCypressVersion() error {
```

Comment:
Name is somewhat misleading. Method suggests that only a check (usually associated with a read-only operation) is performed, yet it's actually also setting values.

Reply:
Renamed it the following:
```
// defineDockerImage defines docker image value if not already set.
func (r *Runner) defineDockerImage() error {
```